### DISCUSSION TITLE
The request timeout

### DISCUSSION THREAD

**Discussion by CongminYang** [resteasy/resteasy#3262] [Q&A]
When the network speed is slow, I have set the client response time to be long, but the request still times out, causing the operation to fail. How should I resolve this situation?

**Comment by jamezp** [comment]
Can you give some more details? Do you have a way to reproduce it?

**Reply by CongminYang** [reply]
I have some very persistent HTTP calls and I need results. I don't know how to extend the server-side timeout for these types of calls. I set the timeout of the client request to a long time, but the request still timed out and returned a "timeout exception" to the client after 2-3 minutes.
